component/authenticator/pom.xml

```diff
@@ -96,6 +96,11 @@
             <version>1.6.5</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.orbit.org.owasp.encoder</groupId>
+            <artifactId>encoder</artifactId>
+            <version>${encoder.wso2.version}</version>
```

Comment:
remove version from here and define in parent pom.xml

Reply:
Fixed b45cb8c